ui/desktop/app/templates/scopes/scope/projects/sessions/index.hbs

```diff
@@ -13,110 +13,52 @@
     <p>{{t 'resources.session.description'}}</p>
   </page.header>
 
-  <page.body>
-    <Rose::Toolbar as |toolbar|>
-      <toolbar.primary>
-        <Rose::Form as |form|>
-          {{#let
-            (resource-filter 'scopes.scope.projects.sessions.index' 'status')
-            as |filter|
-          }}
-            <Rose::Dropdown
-              @text={{t (concat 'form.' filter.name '.label')}}
-              @count={{filter.selectedValue.length}}
-              @ignoreClickInside={{true}}
-              as |dropdown|
-            >
-              <form.checkboxGroup
-                @name={{filter.name}}
-                @items={{filter.allowedValues}}
-                @selectedItems={{filter.selectedValue}}
-                @onChange={{route-action 'filterBy' filter.name}}
-                as |group|
+  <page.body class='search-filtering'>
+    {{#if this.sortedSessions}}
+      <Hds::SegmentedGroup as |S|>
+        <S.Generic>
+          <Dropdown
```

Comment:
I don't think I see anything that would necessitate building your own options, I think this could just be:
```
          <Dropdown
            @name={{t 'resources.target.title'}}
            @itemOptions={{this.availableTargets}}
            @checkedItems={{this.targets}}
            @applyFilter={{fn this.applyFilter 'targets'}}
            @isSearchable={{true}}
            @listPosition='bottom-left'
          />
```

Since the component handles building the items for you if you provide the item options

Reply:
fixed, thank u!

Reply:
I think you'll need to update the component though since it assumes `displayName` and change it in a few places, we should just manually do the `name || id` display rather than assume it's a model with a `displayName`